README.md

```diff
@@ -268,6 +268,22 @@ if err := k.Load(s3.Provider(s3.Config{
 }
 ```
 
+### Reading from an AWS Secrets Manager
```

Comment:
Hey, can you please also add it to https://github.com/knadh/koanf#bundled-providers